pwa-devdocs/src/tutorials/intercept-a-target/change-static-assets-directory/index.md

```diff
@@ -0,0 +1,222 @@
+
+```diff
+        new HTMLWebpackPlugin({
+            filename: 'index.html',
+            template: './template.html',
+            minify: {
+                collapseWhitespace: true,
+                removeComments: true
+            }
+        }),
++       new CopyPlugin({
++           patterns: [{ from: 'my-static-assets', to: 'my-static-assets' }]
++       })
+    ];
+```
+
+This entry tells Webpack to copy the contents of the `my-static-assets` directory into a new `my-static-assets` directory in the `dist` folder during the build process.
+
+## Modify UPWARD configuration
+
+Now that the `my-static-assets` directory is inside `dist`, browsers can access these files under the `/my-static-assets` path.
+However, some files, such as `robots.txt`, `manifest.json`, and `favicon.ico`, must be located at the root.
+To serve these files from the root, you must configure the UPWARD server to look in the `my-static-assets` directory when it gets a request from these files.
+
+### Identify current rule
+
+The [`veniaResponse` object][] in the base `upward.yml` file contains the [conditional resolver][] that tells the server to resolve requests for these files using the `staticFromRoot` object.
+According to this object, the current rule for resolving requests for these files is to look for and return them from the `venia-static` directory.
+
+[`veniaResponse` object]: https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/upward.yml
+[conditional resolver]: https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/upward.yml
+
+### Update UPWARD configuration using targets
+
+Since this rule is inside the primary `upward.yml` file in the `venia-ui` dependency, you cannot make direct edits to this file.
+However, you can use the [`transformUpward` Buildpack target][] to make changes indirectly during the build process.
+
+[`transformUpward` Buildpack target]: <{% link pwa-buildpack/reference/targets/index.md %}>
+
+Open your project's `local-intercept.js` file and replace the content with the following:
+
+```js
+function localIntercept(targets) {
+    targets.of('@magento/pwa-buildpack').transformUpward.tap(def => {
+        def.staticFromRoot.inline.body.file.template.inline =
+            './my-static-assets/{{ filename }}';
+    });
+}
+
+module.exports = localIntercept;
+```
+
+The `transformUpward` target converts the base `upward.yml` file into a JavaScript object you can modify using an interceptor function.
+The interceptor function in the code sample provided sets the inline mustache template value with one that refers to the new static assets directory.
+
+The build process generates the final `upward.yml` configuration under the `dist` directory, and the contents of this file is a combination of:
+
+- Content in the base `upward.yml` file in the `venia-ui` package
+- Modifications to that file using the `transformUpward` target
+- Content in your project's `upward.yml` file
+
+## Update HTML template
```

Comment:
Would it be better to colocate this with the other `Update manifest file` section or in a more general "update `venia-static` references" section?